go.mod

```diff
@@ -1,8 +1,11 @@
 module github.com/coder/terraform-provider-coder
 
-go 1.20
+go 1.21
+
+toolchain go1.22.3
```

Comment:
This was not a deliberate change and is likely due to importing `docker/docker`.